modules/nickserv.cpp

```diff
@@ -119,9 +141,18 @@ class CNickServ : public CModule {
             MCString msValues;
             msValues["password"] = GetNV("Password");
             PutIRC(CString::NamedFormat(GetNV("IdentifyCmd"), msValues));
+        } else if(Nick.NickEquals(sNickServName) && sMessage.find("Password accepted") != CString::npos && m_bJoinAfterIdentified) {
```

Comment:
Sorry, I didn't notice your comment till now. I added support for atheme, but atheme and anope are the only IRC services I know of. Are there any other I should support?